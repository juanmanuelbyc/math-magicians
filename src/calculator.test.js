'use strict'

import React from 'react';
import Calculator from './pages/Calculator.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
   <Calculator />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
