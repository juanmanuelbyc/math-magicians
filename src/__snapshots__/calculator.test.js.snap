// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="calculatorPageContainer"
>
  <h2>
    Lets do some math!
  </h2>
  <div
    className="calculatorContainer"
  >
    <div
      className="display"
    >
      <h1 />
      <h1 />
      <h1 />
    </div>
    <div
      className="buttonsContainer"
    >
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="s1 operationButton button"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="s1 operationButton button"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="s1 operationButton button"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="s1 operationButton button"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="s2 characterButton button"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="s1 characterButton button"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="s1 operationButton button"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
